web/views.py

```
@@ -7103,3 +7103,26 @@ def users_list(request: HttpRequest) -> HttpResponse:
+
+@require_POST
+def free_checkout(request):
+    try:
+        cart = Cart.objects.get(user=request.user)
+        if cart.total_amount() > 0:
+            return JsonResponse({"error": "Cart is not free"}, status=400)
+
+        for item in cart.items.all():
+            Enrollment.objects.get_or_create(
+                user=request.user,
+                course=item.course  # Adjust field name as necessary
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)